sql_request_abstract/README.rst

```diff
@@ -0,0 +1,93 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================================
+Abstract Model to manage SQL Requests
+=====================================
+
+This module provide an abstract model to manage SQL Select request on database.
```

Comment:
s /provide/provides
s /request/requests/

Reply:
Thank for reviewing, comments attended.